MonkeyWrench.Web.UI/EditHost.aspx.cs

```diff
@@ -78,22 +82,26 @@ protected override void OnInit (EventArgs e)
 			}
 
 			if (!string.IsNullOrEmpty (disable) && int.TryParse (disable, out id)) {
-				Utils.LocalWebService.SwitchHostEnabledForLane (Master.WebServiceLogin, id, response.Host.id);
+				DBLane lane = Utils.LocalWebService.FindLane (Master.WebServiceLogin, id, null).lane;

```

Comment:
Here you fetch the lane using the id, just to only use lane.id the next line. That looks redundant.